libs/core/src/lib/pagination/pagination.component.ts

```diff
@@ -270,6 +254,48 @@ export class PaginationComponent implements OnChanges, OnInit, OnDestroy {
         return this._rtlService?.rtl.value;
     }
 
+    /**
+     * Label for the 'Page ' page button. Page number passed as function parameter.
+     * This property is mainly provided to support reading in the right language for screen reader.
+     */
+    @Input()
+    pageLabel: (page: number) => string = (page: number) => `Page ${page}`;
```

Comment:
I'm trying to imaging the main usage of this...
The functional approach pushes a user to keep this function somewhere in his component and use some translation services for it.
As for me it would be much easier to keep this sort of inputs somewhere in the template, so-called declarative approch. In this case user can derive this as a template reference where template itself can provide a page number or whatever is needed for this.
For example:
```
<fd-pagination [pageLabel]="pageLabelTemplateRef" />
<ng-template let-pageNumber="pageNumber" i18n="@@myPagePagination" #pageLabelTemplateRef>
  Page {{pageNumber}}
</ng-template>
```
Does it make sense?

Reply:
It makes sense, i totally agree with you, but exactly this input used for `[attr.aria-label]` so it cannot be done.

From all of the inputs of this component that are used for labels\aria, only one can be provided in the way proposed by you, so i'm not sure to have both approaches in the component - functions & templates. What do you think?